Toolchain.mk

```
@@ -99,13 +104,18 @@ $(HELM): | $(BIN)
 	mv $(BIN)/helm $(HELM) ;\
 	}
 
-HELM_DOCS := $(BIN)/helm-docs-$(HELM_DOCS_VERSION)
+HELM_DOCS := $(BIN)/helm-docs-v$(HELM_DOCS_VERSION)
 $(HELM_DOCS): | $(BIN)
 	$(info $(M) installing helm-docs)
 	@{ \
 	set -e ;\
-	GOBIN=$(BIN) go install github.com/norwoodj/helm-docs/cmd/helm-docs@$(HELM_DOCS_VERSION) ;\
-	mv $(BIN)/helm-docs $(HELM_DOCS) ;\
+	OSTYPE=$(shell uname | awk '{print tolower($$0)}') && ARCH=$(shell go env GOARCH) && \
```

```suggestion
	OSTYPE=$(shell uname) && ARCH=$(shell go env GOARCH) && \
```
The same applies here.